CONTRIBUTING.md

```diff
@@ -0,0 +1,211 @@
+# Contributing to dosbox-staging
+
+Thank you for your interest in contributing to dosbox-staging! There are many
+ways to contribute, and we appreciate all of them. This document is a bit long,
+so here's links to the major sections:
```

Comment:
here are
(here's links -> here is links)

Reply:
I also think we have an opportunity to include (in small comments sprinkled throughout at relevant points) some of your vision and goals (ie: things that you want to different versus upstream), that may interest someone in become part of the community. 
So for example, in the "Tools" section, you could introduce it with a brief sentence about using the best-in-class, modern tooling to improve the code quality and catch more issues.   Same with code reviews - you could state the reason and benefits of that. Just little selling points.